packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -65,50 +67,51 @@ export default class QuanticSearchBoxInput extends LightningElement {
    */
   @api suggestions = [];
   /**
-   * Returns and set the input value.
+   * The value of the input.
    * @api
-   * @type {string}
+   * @type {String}
    */
-  @api
-  get inputValue() {
-    return this.input.value;
-  }
-  set inputValue(newValue) {
-    this.input.value = newValue;
-  }
```

Comment:
Yes if it was an exposed component, it would be a breaking change. but this component was created just to reduce the complexity of the main QuanticSearchBox and is purely internal.

The current logic we have been following lately is that these internal component are purely for us to better organize  the code and the component composition so changing them would not be considered breaking.

Other wise this would limit us a lot and would discourage us to reduce the complexity of the exposed components by using smaller internal component.

and for this change, yeah I changed the name yeah but that's not the main point I wanted to improve here, the main point is that we had a getter and a setter for this property which were useless, just a normal property can do the same job. that's the simplification I'm doing here.

Reply:
It's more that even for an internal component, I wouldn't do a breaking change just to rename a prop for fun.